project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/my_groups/my_groups.inc

```diff
@@ -70,7 +70,7 @@ function c4m_features_homepage_get_user_groups($account = NULL, $allowed_types =
   $groups = c4m_user_og_get_groups($account);
 
   if (empty($groups)) {
-    return FALSE;
+    return array();
```

Comment:
I had to test it, I read first that $usort converts empty arrays to NULL, for that reason I didn't remove it before. But seems that it works normally, if receives an empty array, returns an empty array.